pom.xml

```diff
@@ -44,22 +44,32 @@
         <air.check.skip-spotbugs>true</air.check.skip-spotbugs>
         <air.check.skip-pmd>true</air.check.skip-pmd>
         <air.release.preparation-goals>clean verify -DskipTests</air.release.preparation-goals>
+        <dep.jetty.version>12.0.9</dep.jetty.version>
     </properties>
 
     <dependencyManagement>
         <dependencies>
+
             <dependency>
                 <groupId>io.airlift</groupId>
                 <artifactId>bom</artifactId>
-                <version>240</version>
+                <version>248</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+
+            <dependency>
+                <groupId>org.eclipse.jetty</groupId>
+                <artifactId>jetty-bom</artifactId>
+                <version>${dep.jetty.version}</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
 
             <dependency>
-                <groupId>io.dropwizard</groupId>
-                <artifactId>dropwizard-dependencies</artifactId>
-                <version>4.0.6</version>
+                <groupId>org.eclipse.jetty.ee10</groupId>
```

Comment:
Why do you need to explicitly define jetty dependencies?

Reply:
Same as above. It doesn't exist in `io.airlift:bom` nor `io.airlift:airbase`.
Also, all the `jetty-ee10-*` dependencies will be removed in #382.
We don't need to worry too much about this.